lib/cylc/cfgspec/suite.py

```diff
@@ -66,6 +67,10 @@ def _coerce_cycleinterval( value, keys, args ):
 
 def _coerce_cycletime( value, keys, args ):
     """Coerce value to a cycle point."""
+    if value == "now":
+        value = datetime.datetime.now().strftime("%Y%m%dT%H%M")
```

Comment:
This should really have a time zone as well (can be found via `cylc.wallclock`). If the suite time zone is different from the local time zone, this won't end up giving what we want.